src/lib/arch/aarch64/InstructionMetadata.cc

```diff
@@ -87,6 +87,9 @@ InstructionMetadata::InstructionMetadata(const cs_insn& insn)
       // adds incorrectly flags destination as READ
       operands[0].access = CS_AC_WRITE;
       break;
+    case Opcode::AArch64_BICv8i16:
+      operands[0].access = CS_AC_WRITE | CS_AC_READ;
+      break;
```

Comment:
If access types are not properly set by Capstone decode, you should define them for all operands, not just the destination.